geniusyield-market-maker/test/Main.hs

```diff
@@ -0,0 +1,73 @@
+module Main (main) where
+
+import           Control.Concurrent.MVar
+import           Control.Exception                         (throwIO)
+import           GeniusYield.Api.Dex.Constants             (dexInfoDefaultMainnet,
+                                                            dexInfoDefaultPreprod)
+import           GeniusYield.GYConfig
+import           GeniusYield.MarketMaker.Constants         (logNS)
+import           GeniusYield.MarketMaker.MakerBotConfig
+import           GeniusYield.MarketMaker.MakerBot          (MakerBot(..), cancelAllOrders')
+import           GeniusYield.MarketMaker.Prices
+import           GeniusYield.MarketMaker.Strategies
+import           GeniusYield.MarketMaker.Utils             (addrUser)
+import           GeniusYield.Test.MarketMaker.MakerBot
+import           GeniusYield.Test.MarketMaker.Utils
+import           GeniusYield.OrderBot.DataSource.Providers (connectDB)
+import           GeniusYield.Types                         (GYNetworkId (..), addressToText,
+                                                            gyLog', GYLog(..))
+import           GeniusYield.Types.Providers               (gyLogInfo)
+import           Test.Tasty
+import           Test.Tasty.HUnit
+
+
+-----------------------------------------------------------------------
+----------------------------- MAIN ------------------------------------
+
+runSequence :: IO Bool
+runSequence = do
+  logRef  <- newMVar []
+
+  let action           = "Run"
+  let frameworkCfgPath = "../secrets/my-atlas-config-maestro.json"
```

Comment:
Preferably make it an environment variable (which can be obtained using `getEnv`).

Reply:
Likewise for `mBotConfigFile`.

Reply:
Done.